terminal_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package ansiterm
+
+import (
+	"io/ioutil"
+	"os"
+
+	gc "gopkg.in/check.v1"
+)
+
+type colorWriterSuite struct{}
+
+var _ = gc.Suite(&colorWriterSuite{})
+
+func (s *colorWriterSuite) TestNoColor(c *gc.C) {
+	file, err := ioutil.TempFile("", "")
+	c.Assert(err, gc.IsNil)
+
+	os.Setenv("NO_COLOR", "")
```

Comment:
should we be deferring something to restore NO_COLOR to its original value? Otherwise parts of the test suite will be running in a different context than the rest.